### ISSUE TITLE
provide feedback when the application is doing background stuff

### ISSUE THREAD

**Issue by larntz** [larntz/artui#19]
create a status line or something that shows when an application is syncing or something is happening in the background like reconnecting to argocd if the connection is broken.

**Comment by cshintov** [comment]
Can I work on this? You still using this?

**Comment by larntz** [comment]
No, I haven't worked on this in a while. Changed jobs and I'm no longer using ArgoCD. 

I probably should have archived the repo. If you want to work on it feel free to clone it and I'll archive this repo and link to yours.
